content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -192,6 +192,9 @@ request headers as follows:
   </tbody>
 </table>
 
+{{< warning >}}
+Please note the validation rules within OpenAPI schemas are incomplete and additional validation occurs in the code of the kube-apiserver. If precise and complete verification is required,  a `kubectl apply --dry-run` runs all the applicable validation.
```

Comment:
I think this is correct, and it's more aligned with our style guide.
```suggestion
The validation rules published as part of OpenAPI schemas may not be complete.
Additional validation occurs within the API server. If you want precise and complete verification,
a `kubectl apply --dry-run=server` runs all the applicable validation (and also activates admission-time
checks).
```

Reply:
Tim's suggestion is correct on the technical details.  Thanks.